executor/slow_query.go

```diff
@@ -77,9 +88,35 @@ func (e *SlowQueryRetriever) retrieve(ctx context.Context, sctx sessionctx.Conte
 	return retRows, nil
 }
 
+// Initialize is exported for test.
+func (e *SlowQueryRetriever) Initialize(sctx sessionctx.Context) error {
```

Comment:
Does this method need to be an exported method?

Reply:
You can add an exported method for `SlowQueryRetriever` in `executor_pkg_test.go`. see: https://github.com/pingcap/tidb/blob/9fbefc50b56d5c6d3c9f071965c0451e16243226/executor/executor_pkg_test.go#L39

Reply:
It is exported for the unit-test.